test/distributed/tensor/parallel/test_tp_examples.py

```diff
@@ -107,12 +108,38 @@ def _test_mlp_magatron_e2e(self, is_seq_parallel=False, recompute_activation=Fal
         output_tp = model_tp(inp)
         self.assertEqual(output, output_tp)
 
+    def _test_mlp_magatron_inference(self):
```

Comment:
nit: we should not mentione megatron here, let's just call it sth like test_inference_mode_inference

Reply:
Removed megatron from test name.